hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAdmin2.java

```diff
@@ -300,8 +300,9 @@ public void testCloseRegionIfInvalidRegionNameIsPassed() throws Exception {
           info = regionInfo;
           try {
             ADMIN.unassign(Bytes.toBytes("sample"), true);
-          } catch (UnknownRegionException nsre) {
-            // expected, ignore it
+          } catch (IllegalArgumentException iae) {
```

Comment:
I think here we want to test UnknownRegionException, not IllegalArgumentException? So I think we should change 'sample' to a valid but not exists region name.

Reply:
ok.